src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerTest.java

```diff
@@ -192,9 +191,7 @@ public void should_return_a_200_with_search_results_with_restricted_role_assignm
 
         result.then().assertThat()
             .statusCode(HttpStatus.OK.value())
-            .body("tasks.jurisdiction", everyItem(is("SSCS")))
-            .body("tasks.case_id", hasItem(caseId))
-            .body("tasks.id", hasItem(taskId));
+            .body("tasks", empty());
```

Comment:
I don't think this is correct. 

This test was disabled because it is just not possible to do through functional tests unless we create a real SSCS case, due to the task configuration configuring the task and adding the correct jurisdiction and case type, Hence it must be tested as part of an integration test.

So the assertion is wrong here the one that was disabled previously is the correct one however if this particular scenario has been covered by an integration tests then we can just remove this functional test.

Reply:
The TaskSearch does not throw any 403 if there is any mismatch in permissions instead it throws empty list. 
I agree on creating a case with SCSS if its not supported then it should not be part of any FT.